modules/persistent-storage-csi-vol-detach-non-graceful-shutdown-overview.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vol-detach-non-graceful-shutdown.adoc
+//
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-vol-detach-non-graceful-overview_{context}"]
+= Overview
+
+A graceful node shutdown occurs when the kubelet's node shutdown manager detects the upcoming node shutdown action. Non-graceful shutdowns occur when the kubelet does not detect a node shutdown action. This can happen when the shutdown command does not trigger the Inhibitor Locks mechanism used by the kubelet on Linux, or because of a user error, for example, if the shutdownGracePeriod and shutdownGracePeriodCriticalPods details are not configured correctly for that node.
```

Comment:
That's correct but I think non graceful mostly happen because of system or hardware failures

Reply:
Rewrote. PTAL.